doc/060_forget.rst

```diff
@@ -189,7 +189,19 @@ The ``forget`` command accepts the following parameters:
    the duration of the latest snapshot. ``duration`` needs to be a number of
    years, months, days, and hours, e.g. ``2y5m7d3h`` will keep all snapshots
    made in the two years, five months, seven days, and three hours before the
-   latest snapshot.
+   latest snapshot`
```

Comment:
That sentence should end with a `.` and also the new item.

Reply:
Should be fixed now.